daprdocs/content/en/developing-applications/building-blocks/actors/actor-reentrancy.md

```diff
@@ -20,25 +16,15 @@ ActorA -> Actor B -> Actor A
 
 With reentrancy, there can be more complex actor calls without sacrificing the single-threaded behavior of virtual actors.
 
-## Enabling actor reentrancy
-Actor reentrancy is currently in preview, so enabling it is a two step process.
-
-### Preview feature configuration
-Before using reentrancy, the feature must be enabled in Dapr. For more information on preview configurations, see [the full guide on opting into preview features in Dapr]({{< ref preview-features.md >}}). Below is an example of the configuration for actor reentrancy:
-
-```yaml
-apiVersion: dapr.io/v1alpha1
-kind: Configuration
-metadata:
-  name: reentrantconfig
-spec:
-  features:
-    - name: Actor.Reentrancy
-      enabled: true
-```
+The `maxStackDepth` parameter sets a value that controls how many reentrant calls be made to the same actor. By default this is set to 32, which is more than sufficient in most cases.
+
+## Enable Actor Reentrancy with Actor Configuration
 
-### Actor runtime configuration
-Once actor reentrancy is enabled as an opt-in preview feature, the actor that will be reentrant must also provide the appropriate configuration to use reentrancy. This is done by the actor's endpoint for `GET /dapr/config`, similar to other actor configuration elements. Here is a snipet of an actor written in Golang providing the configuration:
+The actor that will be reentrant must provide the appropriate configuration to use reentrancy. This is done by the actor's endpoint for `GET /dapr/config`, similar to other actor configuration elements. Here is a snipet of an actor written in Golang providing the configuration:
+
+{{< tabs Go Dotnet Python JavaScript >}}
```

Comment:
```suggestion
{{< tabs Dotnet Java Python Go JavaScript >}}
```
Put the code into this order please and also add a blank entry for Java, with a sentence saying "Reentrancy has not yet been included into the Java SDK"